Sources/TensorFlow/Core/MixedPrecision.swift

```diff
@@ -157,7 +157,7 @@ extension Tensor {
   }
 
   /// Promotes a scalar to a tensor with the same device and precision as the given tensor.
-  @differentiable(where Scalar: TensorFlowFloatingPoint)
+  @differentiable( where Scalar: TensorFlowFloatingPoint)
```

Comment:
Note: seems like it's mis-formatting this at the moment. (I think you're already aware, but wanted to make sure to flag them here.)

Reply:
Yeah, @allevato is going to be looking into backporting the fixes for that to the 5.2 release so we can get these correct.